packages/typespec-azure-playground-website/samples/arm.tsp

```diff
@@ -55,20 +55,21 @@ model EmployeeProperties {
 
 /** The provisioning state of a resource. */
 @lroStatus
-enum ProvisioningState {
-  ...ResourceProvisioningState,
+union ProvisioningState {
+  string,
+  ResourceProvisioningState,
```

Comment:
I think here we should just add in the values from the standard provisioning state, since this is probably not what we want
```suggestion
  /** Resource has been created.*/
  Succeeded: "Succeeded",

  /** Resource creation failed.*/
  Failed: "Failed",

   /** Resource creation was canceled.*/
  Canceled: "Canceled",
```